client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -1,104 +1,127 @@
-<section class="telephony-group-alias-configuration-change-type">
+<div class="text-center"
+     data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
 
-    <header>
-        <section-back-link section-back-link-to-state="telecom.telephony.alias.configuration"
-                           section-back-link-title="{{ 'telephony_alias_configuration_back_link' | translate }}">
-        </section-back-link>
-        <h1 data-translate="telephony_alias_change_type_title"></h1>
+<div class="container-fluid ml-0"
+     data-ng-if="!$ctrl.loading">
+    <toast-message></toast-message>
 
-    </header>
-
-    <div class="text-center" data-ng-if="AliasConfigurationChangeTypeCtrl.loading.line">
-        <oui-spinner></oui-spinner>
+    <div class="row">
+        <h2 class="col-lg-5 col-sm-12 oui-header_2"
+            data-translate="telephony_alias_change_type_title"></h2>
     </div>
 
-    <div class="telecom-section-content"
-         data-ng-if="!AliasConfigurationChangeTypeCtrl.loading.line">
+    <div class="row">
+        <oui-collapsible class="col-lg-8 col-sm-12 ml-0 mr-2 mb-4"
```

Comment:
`col-sm-12` is useless.

⚠️ `ml-0 mr-2` aren't compatible with `col-lg-8`.

Reply:
`col-sm-12` is useless indeed. But `ml-0 mr-2` are compatible with `col-lg-8`
(In my case `mr-2` is useless, so I'll keep only `ml-0`)